app/services/staff_nomis_checker.rb

```diff
@@ -72,8 +73,27 @@ def slots_unavailable?
     end
   end
 
+  def no_allowance?
```

Comment:
Doesn't seem a good idea as they refer a lot to the slot errors stored locally - if they were moved to `RejectionDecorator` they would have to reach across the object model and break the Law of Dementer.